addons/project/models/project.py

```diff
@@ -336,20 +346,20 @@ def toggle_favorite(self):
         favorite_projects.write({'favorite_user_ids': [(3, self.env.uid)]})
 
     @api.multi
-    def close_dialog(self):
-        return {'type': 'ir.actions.act_window_close'}
-
-    @api.multi
-    def edit_dialog(self):
-        form_view = self.env.ref('project.edit_project')
+    def open_tasks(self):
+        ctx = dict(self._context)
+        ctx.update({'search_default_project_id': self.id})
+        kanban_view_id = self.env.ref('project.view_task_kanban')
         return {
-            'name': _('Project'),
-            'res_model': 'project.project',
-            'res_id': self.id,
-            'views': [(form_view.id, 'form'),],
-            'type': 'ir.actions.act_window',
-            'target': 'inline'
-        }
+                'name': _('Tasks'),
```

Comment:
can you take this text name from the project configuration itself, (Tasks or what the project itself says).